dir.targets

```diff
@@ -17,7 +17,30 @@
 
             var tempFile = Path.Combine(directory, Path.GetRandomFileName());
             var client = new System.Net.WebClient();
-            client.DownloadFile(Address, tempFile);
+            var tryCount = 1;
+            var maxTries = 3;
+
+            while (tryCount <= maxTries)
+            {
+                try
+                {
+                    Log.LogMessage("Attempting to download {0}...", Address);
+                    client.DownloadFile(Address, tempFile);
+                    break;
+                }
+                catch (System.Net.WebException e)
```

Comment:
Do we always want to retry like this, or do e need to distinguish between cases where the server said the file wasn't available vs the server couldn't be reached/etc.?